.github/actions/setup-pyenv/action.yml

```diff
@@ -0,0 +1,52 @@
+name: "setup-pyenv"
+description: "Setup pyenv"
+runs:
+  using: "composite"
+  steps:
+    ########## Ubuntu ##########
+    - name: Install python build tools
+      if: runner.os == 'Linux'
+      shell: bash
+      # Ref: https://github.com/pyenv/pyenv/wiki#suggested-build-environment
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y make build-essential libssl-dev zlib1g-dev \
+        libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+        libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+    - name: Install pyenv
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        git clone --depth 1 https://github.com/pyenv/pyenv.git $HOME/.pyenv
+    - name: Setup environment variables
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        PYENV_ROOT="$HOME/.pyenv"
+        PYENV_BIN="$PYENV_ROOT/bin"
+        echo "$PYENV_BIN" >> $GITHUB_PATH
+        echo "PYENV_ROOT=$PYENV_ROOT" >> $GITHUB_ENV
+    - name: Check pyenv version
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        pyenv --version
+
+    ########## Windows ##########
+    - name: Install pyenv
+      if: runner.os == 'Windows'
+      shell: bash
+      run: |
+        pip install pyenv-win --target $HOME\\.pyenv
+    - name: Setup environment variables
+      if: runner.os == 'Windows'
+      shell: bash
+      run: |
+        echo "PYENV=$USERPROFILE\.pyenv\pyenv-win\\" >> $GITHUB_ENV
+        echo "PYENV_HOME=$USERPROFILE\.pyenv\pyenv-win\\" >> $GITHUB_ENV
+        echo "$USERPROFILE\.pyenv\pyenv-win\\bin\\" >> $GITHUB_PATH
+    - name: Check pyenv version
+      if: runner.os == 'Windows'
+      shell: bash
+      run: |
+        pyenv --version
```

Comment:
pyenv is successfully installed:
https://github.com/mlflow/mlflow/runs/5537294387?check_suite_focus=true#step:4:35

Reply:
In windows is it ok that MLFLOW_CONDA_HOME is specified as a linux fs path? 

Run pyenv --version
  pyenv --version
  shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
  env:
    **MLFLOW_CONDA_HOME: /usr/share/miniconda**
    pythonLocation: C:\hostedtoolcache\windows\Python\3.7.9\x64
    PYENV: C:\Users\runneradmin\.pyenv\pyenv-win\
    PYENV_HOME: C:\Users\runneradmin\.pyenv\pyenv-win\
pyenv 2.64.11

Reply:
Good catch, no it's not. 

https://github.com/actions/virtual-environments/blob/b7f276c003aea42575b52247bdb2183e355fca2f/images/win/scripts/Installers/Install-Miniconda.ps1#L6

```
$CondaDestination = "C:\Miniconda"
```

In windows, miniconda is installed in `C:\Miniconda"`.